Dockerfile

```diff
@@ -1,36 +1,68 @@
-FROM ruby:2.4.5-slim-stretch
+# XXX: using stretch here for pdftk dep, which is not availible after
+#      stretch (or in alpine) and is switched automatically to pdftk-java in buster
+#      https://github.com/department-of-veterans-affairs/va.gov-team/issues/3032
+
+###
+# shared configs for all child images, reuse these layers yo
+###
+FROM ruby:2.4.5-slim-stretch AS base
+
+ARG userid=993
+RUN groupadd -g $userid -r vets-api && \
+    useradd -u $userid -r -m -d /srv/vets-api -g vets-api vets-api
+RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y \
```

Comment:
Won't this give us potentially different versions of `clamav` etc every time? Can we move that higher up in a build (perfect world would be in the `FROM`) so we get repeatable versions? 

I realize this might be a further down the road issue. It doesn't make anything appreciably worse to leave it as-is since that's our current state.

Reply:
You're correct in that this is no different that what is done now (either here or in the current vets-api BRD image builds) so I don't think this is worth dealing with here and is outside the scope of my current task.

The source distro is still Debian which is conservative with updates and compatibility so this shouldn't be a big issue.